tez-dag/src/main/java/org/apache/tez/dag/app/dag/Task.java

```diff
@@ -21,7 +21,9 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
```

Comment:
the changes (two imports) in this file do not seem necessary

Reply:
removed